website/content/blog/2023-10-kubecon-chicago.md

```diff
@@ -0,0 +1,59 @@
+---
+title:  TAG App Delivery at Kubecon Chicago
+slug:   tag-app-delivery-at-kubecon-chicago
+date:   2023-10-09 12:00:00 +0000
+author: Josh Gavant
+---
+
+![Kubecon Chicago 2023](/images/kubecon-chicago-2023.jpg)
+
+At Kubecon Chicago TAG App Delivery will bring together maintainers and users of
+projects that enable cloud-native application delivery to meet and learn from
+each other. The TAG's goal as always is to enable CNCF's app delivery projects
```

Comment:
@joshgav suggested wording: 
The TAG's goals, as always, are to:
- enable application delivery projects to learn from each other as well as from cloud application developers
- make application delivery faster and more efficient for end users.

( suggest to remove CNCF app delivery projects since they don't all belong to CNCF, e.g. sigstore )